src/In.ProjectEKA.HipService/DataFlow/DataEntryFactory.cs

```diff
@@ -71,7 +72,8 @@ public DataEntryFactory()
                 });
             }
 
-            var keyStructure = new KeyStructure(DateTime.Now.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ"),
+            var dto = DateTimeOffset.Now;
```

Comment:
Do we need to have local variable dto? can we directly use DateTimeOffset.Now.LocalDateTime only?

Reply:
yes, will change